### ISSUE TITLE
Add new release

### ISSUE THREAD

**Issue by iloveitaly** [dim-an/cod#42]
Adding a new release would allow a homebrew config to automatically install a more recent version.

Here's the PR on homebrew https://github.com/Homebrew/homebrew-core/pull/75589

**Comment by dim-an** [comment]
Hi.

I planned to use Circle CI to build releases, but when they closed free plan I had to move to github actions.
I've set up running tests but didn't set up pushing releases.

I'll try to make required setup in next few days and create a new release.

**Comment by iloveitaly** [comment]
Sounds great! Mind pinging me here when you get that running? I'll finish up the HomeBrew PR when that's done.

**Comment by dim-an** [comment]
Sure!

**Comment by dim-an** [comment]
Creating releases from GitHub Actions appeared harder than I thought, I spent few evenings and didn't manage to setup it yet.

I don't have much time now (and finishing CI setup can take few more weeks) so I created new release manually.

Unfortunately I don't have MacOS machine (I used golang's crosscompilation to get darwin binary).
I would appreciate if you test that darwin binary is ok (I already tested that it doesn't work on Linux :smile: )
